server/src/database/notes_observer.go

```diff
@@ -31,6 +32,9 @@ func (*NoteUpdate) AfterUpdate(ctx context.Context, _ *bun.UpdateQuery) error {
 
 func (*Note) AfterDelete(ctx context.Context, _ *bun.DeleteQuery) error {
 	result := ctx.Value("Result").(*[]Note)
+	//TODO: works but how to notify about UpdatedNotes in the right way
+	notifyNotesUpdated(ctx)
```

Comment:
Is this the right way to notify frontend about the update of stack references of the children notes when the overall db function is actually a delete operation that triggers only the afterDelete function?